helm/monitoring/values.yaml

```diff
@@ -0,0 +1,239 @@
+prometheus:
+
+  serverFiles:
+    alerting_rules.yml:
+      groups:
+
+      - name: Nodes
+        rules:
+
+        - alert: NodeMemoryWarn
+          expr: node_memory_MemAvailable_bytes / node_memory_MemTotal_bytes * 100 < 20
+          for: 5m
+          labels:
+            severity: warning
+          annotations:
+            identifier: "{{ $labels.kubernetes_node }}"
+            description: "Low memory on node ({{ $value }}% available)"
+
+        - alert: NodeMemoryCritical
+          expr: node_memory_MemAvailable_bytes / node_memory_MemTotal_bytes * 100 < 10
+          for: 5m
+          labels:
+            severity: critical
+          annotations:
+            identifier: "{{ $labels.kubernetes_node }}"
+            description: "Low memory on node ({{ $value }}% available)"
+
+        - alert: NodeDiskSpaceWarn
+          expr: (node_filesystem_used_bytes / node_filesystem_size_bytes) * 100 < 20
+          for: 5m
+          labels:
+            severity: warning
+          annotations:
+            identifier: "{{ $labels.kubernetes_node }}"
+            description: "Low disk space on node ({{ $value }}% available)"
+
+        - alert: NodeDiskSpaceCritical
```

Comment:
Is filesystem size the amount of space available? If so, if you had 4GB allocated for Node and 3.7 was being used, wouldn't you get 3.7/4 = 92.5% is in use, which is less than 10% free? But then you'd get 9.25 > 10 and the alert wouldn't trigger, even though it should.

Reply:
Good catch - will fix